configure.py

```diff
@@ -1343,7 +1343,7 @@ def main():
   # environment variables.
   environ_cp = dict(os.environ)
 
-  check_bazel_version('0.5.4')
+  check_bazel_version('0.10.0')
```

Comment:
We should fix the build issues with 0.10 in old branches and then make this change.
We will probably take care of all the cherrypicks later this week.

Reply:
We also need to bump the bazel version on all build infra and all our docker images before we do this.

Reply:
Sure, we can wait for them.